addons/account/views/res_config_settings_views.xml

```diff
@@ -184,6 +184,17 @@
                                 <field name="display_invoice_tax_company_currency"/>
                             </setting>
                         </block>
+                        <block title="Units of Measure">
+                            <setting id="account_uom" help="Sell and purchase products in different units of measure">
+                                <field name="group_uom"/>
+                                <div class="content-group">
+                                    <div class="mt8" invisible="not group_uom">
```

Comment:
this div is useless i think we can put the invisible above and the mt8 is not needed, it's overriden by the content group i think